src/chainstate/stacks/index/proofs.rs

```diff
@@ -699,7 +699,7 @@ impl<T: MarfTrieId> TrieMerkleProof<T> {
                 hash_idx += 1;
             }
         }
-        trace!("Shunt proof node: idx={}, all_hashes={:?}", idx, all_hashes);
+        test_debug!("Shunt proof node: idx={}, all_hashes={:?}", idx, all_hashes);
```

Comment:
Please keep the use of `trace!`. You can trigger trace debugging by setting the `STACKS_LOG_TRACE=1` environ.

Reply:
Yeah I was gonna go and revert all of the logging back to original state before an eventual merge, still have a few failing tests I need to figure out then I'll bite that bullet :)